### PR TITLE
browse tool: Allow customization of port number, Py3 fix

### PR DESCRIPTION
**Pull request by Lekensteyn**
Note: the environment variable is named `PORT`, but could also be renamed if you'd like.

I use this tool also remotely over SSH where I set the `BROWSER=true` environment variable to avoid lynx from popping up. Is that information worth adding to the manual (it is part of the Python webbrowser package)?

### CODE REVIEW

File: src/browse.py [hunk 1]
```diff
@@ -31,7 +31,10 @@
 import subprocess
 import sys
 import webbrowser
-import urllib2
+try:
+    from urllib2 import unquote
```
Comment (nico) [inline src/browse.py]: Isn't the recommendation "try py3 variant, use py2 as fallback"? (I'm not sure, I don't do much py3.)

**Comment by nico** [comment]
Thanks! What do you think about making the port a command-line parameter for the tool instead of an env var?

File: src/browse.py (hunk 1 above)
Comment (Lekensteyn) [inline src/browse.py]: Yes it is, fixed in updated patch.

**Comment by Lekensteyn** [comment]
The tool is normally invoked via "ninja -t browse [target name to investigate]". I would be ok with (in fact, prefer) adding something like `ninja -t browse -p 1234`, is this something that aligns with the ninja command options?

While at it, I would add a `--no-browser` option as well (inspired by LLVM's `scan-view` tool).

**Comment by Lekensteyn** [comment]
I went ahead and replaced the PORT env var with --port and also added a --no-browser option. The initial command is also made optional (using `all` instead as default, matches the behavior of invoking `ninja` without options).

**Comment by nico** [comment]
Looks great, thanks!

**Pull request merged**